templates/list-action.tmpl

```diff
@@ -0,0 +1,12 @@
+{% extends 'question.html' %}
+
+{% block cancel_and_return %}
+  {% if previous_location_url %}
+    <p class="u-mt-l">
+      {% if content.block.cancel_text %}
+        {{ content.block.cancel_text }}
```

Comment:
The pull request looks good. We just need to check with @charlotte-hirst if there should be a line break here so it's consistent with the return to hub link.

Reply:
Checked with @charlotte-hirst, have now added a line break to make it consistent with the return to hub link.